src/additional-checks.js

```diff
@@ -8,24 +8,6 @@ import {Job} from "./job.js";
  * @return {Boolean}
  */
 const validateCheck = function (checkConfig) {
-    if (typeof checkConfig !== 'object' || checkConfig === null) {
-        // NOT AN OBJECT!
-        core.warning("Skipping additional check; not an object, or is null: " + JSON.stringify(checkConfig));
-        return false;
-    }
-
-    if (checkConfig.name === undefined || checkConfig.job === undefined) {
-        // Missing one or more required elements
-        core.warning("Skipping additional check due to missing name or job keys: " + JSON.stringify(checkConfig));
-        return false;
-    }
-
-    if (typeof checkConfig.job !== 'object' || checkConfig.job === null) {
```

Comment:
Shouldn't we validate this? Nothing in the type declaration above says that `checkConfig.job` must be an `object`